packages/circuit-ui/components/Toggle/Toggle.tsx

```diff
@@ -93,13 +97,24 @@ export const Toggle = forwardRef(
 
     const switchId = uniqueId('toggle-switch_');
     const labelId = uniqueId('toggle-label_');
+    const descriptionId = explanation ? uniqueId('toggle-explanation_') : '';
```

Comment:
Let's follow the same pattern as form components here: if a developer passes a custom `aria-describedby` (not that it's very common 🙈) we should not override it internally, but concatenate the internal `descriptionId` with the external custom id

Reply:
Done in 5758d72035b30611fb6a6510599881ed2e156194.